src/main/java/core/basesyntax/service/impl/CsvDataConverterService.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataConverterService;
+import core.basesyntax.service.FileReaderService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvDataConverterService implements DataConverterService {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int NUMBER_OF_FRUITS_INDEX = 2;
+    private static final String DATA_SEPARATOR = ",";
+    private FileReaderService fileReader;
```

Comment:
your services should be independent. 
FileReaderService - read all file data and return list of strings in main, then this service convert data from every string to FruitTransaction and etc.